docs/user_guide/modeling_extensions/uncertainty_propagation/index.rst

```diff
@@ -0,0 +1,125 @@
+===============================
+Uncertainty Propagation Toolbox
+===============================
+The uncertainty_propagation module provides tools for calculating error propagation of the objective function and constraints that include uncertain parameters to be estimated for a given model. 
```

Comment:
Please update to use the alternate text provided in the documentation one page up.